core/server/data/schema/schema.js

```diff
@@ -729,5 +729,11 @@ module.exports = {
         },
         subscribe_on_signup: {type: 'bool', nullable: false, defaultTo: false},
         sort_order: {type: 'integer', nullable: false, unsigned: true, defaultTo: 0}
+    },
+    members_newsletters: {
+        id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+        member_id: {type: 'string', maxlength: 24, nullable: false, references: 'members.id', cascadeDelete: true},
+        newsletter_id: {type: 'string', maxlength: 24, nullable: false, references: 'newsletters.id', cascadeDelete: true},
+        sort_order: {type: 'integer', nullable: false, unsigned: true, defaultTo: 0}
```

Comment:
Why would we need sort order on this? 🤔

Reply:
@ErisDS Yea we don't have a usecase for it rn, had only kept this in to have the same convention as other member/product pivot tables, but guess that's not needed ? Will get rid of it in that case.

Reply:
Updated